doc/ui-testing-with-maestro.md

```diff
@@ -0,0 +1,46 @@
+# UI testing with Maestro
+
+##  Install the Maestro CLI
```

Comment:
Can we add the following things to make it contextual to the project: [Ref](https://github.com/simpledotorg/simple-android/blob/master/doc/arch/011-screen-navigation-v2.md)
1. Adding "Context" (why we need to use Maestro tests in the project)
2. Goal (What are we achieving from Maestro in Simple)

Reply:
You can look at this for example:
https://github.com/simpledotorg/simple-android/blob/master/doc/integration-test-suite-setup.md